cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -733,10 +733,10 @@ public:
     _CCCL_HOST_DEVICE _CCCL_FORCEINLINE ScaleT ComputeScale(int num_levels, __half max_level, __half min_level)
     {
       ScaleT result;
-      NV_IF_TARGET(NV_PROVIDES_SM_53,
-                   (result.reciprocal = __hdiv(static_cast<__half>(num_levels - 1), __hsub(max_level, min_level));),
-                   (result.reciprocal = static_cast<float>(num_levels - 1)
-                                      / (static_cast<float>(max_level) - static_cast<float>(min_level));))
+      NV_IF_TARGET(
+        NV_PROVIDES_SM_53,
+        (result.reciprocal = __hdiv(__half2float(num_levels - 1), __hsub(max_level, min_level));),
+        (result.reciprocal = static_cast<float>(num_levels - 1) / (__half2float(max_level) - __half2float(min_level));))
```

Comment:
The tests fail to compile because no declaration is available for `__half2float`. Maybe an include is missing.

Reply:
Oh that is me switching `__half2float` with `__float2half`